test/Deployment.js

```diff
@@ -0,0 +1,101 @@
+const Contracts = require('./helpers/Contracts');
+const runDeployment = require('./helpers/runDeployment');
+
+const config =
+{
```

Comment:
The bracket should be in the same line. Did you format these files?

Reply:
ah yeah, copy-paste (from the README file) mistake

(yes, I actually did format these files afaik)

Reply:
fixed